src/rpc/rawtransaction.cpp

```diff
@@ -816,10 +816,11 @@ static RPCHelpMan sendrawtransaction()
 {
     return RPCHelpMan{"sendrawtransaction",
                 "\nSubmit a raw transaction (serialized, hex-encoded) to local node and network.\n"
-                "\nNote that the transaction will be sent unconditionally to all peers, so using this\n"
+                "\nThe transaction will be sent unconditionally to all peers, so using sendrawtransaction\n"
                 "for manual rebroadcast may degrade privacy by leaking the transaction's origin, as\n"
                 "nodes will normally not rebroadcast non-wallet transactions already in their mempool.\n"
-                "\nAlso see createrawtransaction and signrawtransactionwithkey calls.\n",
+                "\nRPC_TRANSACTION_ALREADY_IN_CHAIN will be raised if the transaction already exists in a block\n"
```

Comment:
This makes it sound like the only exception that can be thrown

Reply:
We can either document every exception that can be thrown or revert back to specifying that this is a note.
```suggestion
                "\nNOTE: an RPC_TRANSACTION_ALREADY_IN_CHAIN will be raised if the transaction already exists in a block\n"
```

Reply:
I would avoid both adding note this/note that (IIRC I added the first note but have since seen review feedback to avoid needless "note: "s, which I agree with) and listing all the exceptions. Is there a concrete suggestion how this change can be improved?

Reply:
"This may throw if the transaction couldn't be added to the mempool"

Reply:
> "This may throw if the transaction couldn't be added to the mempool"

That still sounds like it's the only exception which may be thrown. It also sounds like it's the only reason that a transaction fails to enter the mempool.

Reply:
As-is seems more explicit but I don't have a strong opinion regarding it versus Marco's suggestion (thanks for the proposal!) I think they are both an improvement and don't see the implication that this the only exception.￼￼

Reply:
addressed in  74d23bf